integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -130,12 +130,32 @@ def export(self, **kwargs):
                             )
                         )
 
-                    content_metadata_item_export = ContentMetadataItemExport(item, transformed, content_last_modified)
+                    content_metadata_item_export = ContentMetadataItemExport(
+                        item,
+                        transformed,
+                        content_last_modified,
+                        enterprise_customer_catalog.uuid
+                    )
                     content_metadata_export[content_metadata_item_export.content_id] = content_metadata_item_export
             else:
                 LOGGER.info(
                     "Skipping export of catalog:{} because no updates are needed.".format(enterprise_customer_catalog)
                 )
+                ContentMetadataItemTransmission = apps.get_model(
+                    'integrated_channel',
+                    'ContentMetadataItemTransmission'
+                )
+                past_catalog_transmission_queryset = ContentMetadataItemTransmission.objects.filter(
+                    enterprise_customer=self.enterprise_configuration.enterprise_customer,
+                    integrated_channel_code=self.enterprise_configuration.channel_code(),
+                    enterprise_customer_catalog_uuid=enterprise_customer_catalog.uuid
+                )
+                for past_transmission in past_catalog_transmission_queryset:
```

Comment:
Might be useful to capture a log entry here -- eg, "Found X past transmission items."